docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -110,6 +110,9 @@ You can also run `kubectl logs <pod_name>` to get logs from any failing pod.
         1. This is usually due to an expired or invalid certificate in the cluster, and if so, renew the certificate.
         2. If the certificate is valid, but is signed by a local CA, make sure you have followed the procedure for a [local certificate authority](../config/org-cert.md).
 
+#### Get Installation Logs
+You can use script in the following [link](https://github.com/run-ai/public/blob/main/installation/get-installation-logs.sh) to obtain any relevant installation logs in case of an error
+
```

Comment:
@roir 
```suggestion
#### Get Installation Logs
You can use the following [script](https://github.com/run-ai/public/blob/main/installation/get-installation-logs.sh) to obtain any relevant installation logs in case of an error.
```